backends/pkg/reconciler/eventmesh/controller.go

```diff
@@ -25,9 +28,22 @@ func startWebServer(ctx context.Context) {
 
 	logger.Infow("Starting eventmesh-backend webserver")
 
-	noTokenConfig, err := rest.InClusterConfig()
-	if err != nil {
-		log.Fatalf("Error getting in-cluster config: %v", err)
+	var noTokenConfig *rest.Config
+	var err error
+
+	// if KUBECONFIG env var is present, use it, instead of in-cluster config
+	// this is especially useful for local development
+	kubeConfigPath := os.Getenv("KUBECONFIG")
+	if kubeConfigPath != "" {
+		noTokenConfig, err = clientcmd.BuildConfigFromFlags("", kubeConfigPath)
+		if err != nil {
+			log.Fatalf("Error building kubeconfig: %v", err)
+		}
+	} else {
+		noTokenConfig, err = rest.InClusterConfig()
+		if err != nil {
+			log.Fatalf("Error getting in-cluster config: %v", err)
+		}
 	}
```

Comment:
I think we can just use

```
injection.ParseAndGetRESTConfigOrDie()
```
and that has support for the stardard way of providing a KubeConfig

Reply:
Hm, nice! Let me try to apply that

Reply:
Done!